main.js

```diff
@@ -21,6 +21,8 @@ var activityError = document.querySelector('.activity-error');
 var minutesError = document.querySelector('.minutes-error');
 var secondsError = document.querySelector('.seconds-error');
 
+var lineError = document.querySelector('.warning-line');
```

Comment:
This query selector is not needed. CSS formatting for the error messages is added/removed using the classList method in `activityErrorMessage()` `minutesErrorMessage()` and `secondsErrorMessage`. I removed this line and committed the change directly to this branch.